aea/protocols/base.py

```diff
@@ -217,8 +180,13 @@ def __eq__(self, other):
         """Compare with another object."""
         return (
             isinstance(other, Message)
+            and self._sender == other._sender
+            and self._to == other._to
+            # and self.dialogue_reference == other.dialogue_reference  # noqa: E800
+            # and self.message_id == other.message_id  # noqa: E800
+            # and self.target == other.target  # noqa: E800
+            # and self.performative == other.performative  # noqa: E800
```

Comment:
Currently, this is not possible. We first need to implement the `message` format in protobuf and add the default fields so we can properly serialise.